pkgs/applications/window-managers/hyprwm/hyprland/default.nix

```diff
@@ -66,6 +67,7 @@ stdenv.mkDerivation (finalAttrs: {
       --replace "@HASH@" '${finalAttrs.src.rev}' \
       --replace "@BRANCH@" "" \
       --replace "@MESSAGE@" "" \
+      --replace "@DATE@" "20231206" \
```

Comment:
My bad, I was not explicit enough. In the flake we use ISO 8601 formatting, so in my case the output looks like `Date: 2023-12-27`.

I think it's more readable this way :)

Reply:
Done! Thanks for your time (: